src/MessageHandler/ActivityPub/Outbox/DeliverHandler.php

```diff
@@ -53,8 +59,21 @@ public function workWrapper(MessageInterface $message): void
             $this->doWork($message);
             $conn->commit();
         } catch (InvalidApPostException $e) {
+            if (400 <= $e->responseCode && 500 > $e->responseCode && 429 !== $e->responseCode) {
+                $conn->rollBack();
```

Comment:
Agreed, however, I would also like to enable debugging logging as an admin.

Allowing to see which servers (domain + response body) are giving me invalid HTTP requests back. And try to understand why.